src/components/Certificate/certificate.tsx

```diff
@@ -0,0 +1,37 @@
+import { useLocation, useParams } from "react-router-dom";
+import { css } from "../../../styled-system/css";
+
+const Certificate = () => {
```

Comment:
```suggestion
export default function Certificate() {
```

화살표 함수에 대한 선호도가 크지 않으시다면 컴포넌트 작성하실 때는 `function` 키워드 쓰시는 것을 추천드리고 싶어요. 컴포넌트의 크기가 커지고 그 안에 내부 함수들이 생겼을 때 가독성이 살짝 더 나아지는 것 같더라고요. 파일을 열자마자 컴포넌트를 후다닥 찾기도 용이하고요.

Reply:
좋은의견이네요. 컴포넌트는 function 키워드로 수정하겠습니다.